karma.conf.js

```diff
@@ -70,7 +69,7 @@ module.exports = function(config) {
     // test results reporter to use
     // possible values: 'dots', 'progress'
```

Comment:
Is this canned documentation?  Should it say "examples:" instead of "possible values:"?

Reply:
think that is canned.